scripts/deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Build
+npm run build-release
+rsync -a release/ example/
+mkdir -p example/jquery/dist
+cp node_modules/jquery/dist/jquery.min.js example/jquery/dist/jquery.min.js
+rm example/server.js
+
+# Publish to gh-pages
+cd example
+git init
+git config user.name "Travis CI"
+git config user.email "charles.rudolph@originate.com"
```

Comment:
Can we create a new cucumber account (unless you have one already) who can be used exclusively for this sort of deployment?